DataFormats/HcalIsolatedTrack/interface/IsolatedL1TPixelTrackCandidate.h

```diff
@@ -0,0 +1,152 @@
+#ifndef HcalIsolatedTrack_IsolatedPixelTrackCandidate_h
+#define HcalIsolatedTrack_IsolatedPixelTrackCandidate_h
+/** \class reco::IsolatedPixelTrackCandidate
+ *
+ *
+ */
+
+#include "DataFormats/RecoCandidate/interface/RecoCandidate.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+
+#include "DataFormats/L1Trigger/interface/Jet.h"
+
+#include "DataFormats/HcalIsolatedTrack/interface/IsolatedPixelTrackCandidateFwd.h"
+
+#include <vector>
+#include <map>
+#include <utility>
+
+namespace reco {
+  
+  class IsolatedPixelTrackCandidate: public RecoCandidate {
```

Comment:
looks like replace step of  IsolatedPixelTrackCandidate -> IsolatedL1TPixelTrackCandidate failed in this file.
Do we really have to generate a new data format for this?